### ISSUE TITLE
Element 84 Earth Search API Consumer Contact Email

### ISSUE THREAD

**Issue by DanSchoppe** [stac-utils/stac-server#105]
Hi again @matthewhanson 😄, this is a different enough topic that I figured it deserved it's own issue.

In #104 you mention:

> I'm not sure yet what this means for the L2A COGs, I'd like to get those updated and processed in a new pipeline, but I have to figure out logistics on how and when to cut over since there's other systems relying on the v0 API (which won't be going away for a while but I need to figure out a transition plan).

That got me thinking: Does Element 84 provide any way for API consumers to let Element 84 know of our existence? It would be nice to provide an email address that could be used to communicate upcoming major changes (v0 -> v1) or to be able to reach out in case queries or usage patterns are causing problems.

**Comment by matthewhanson** [comment]
@DanSchoppe Good idea, I'll look into setting up an email list for announcements and such.  Thanks!
